Source/Core/VideoBackends/D3D12/PerfQuery.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright 2012 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "VideoBackends/D3D12/D3DBase.h"
+#include "VideoBackends/D3D12/PerfQuery.h"
+#include "VideoCommon/RenderBase.h"
+
+namespace DX12
+{
+
+PerfQuery::PerfQuery()
+{
+	for (ActiveQuery& entry : m_query_buffer)
+	{
+#ifdef USE_D3D11
```

Comment:
Is this code meant to be left in? I think it's confusing to have this in the D3D12 tree, and better left out until a D3D12 version is written.